src/libraries/NetCoreAppLibrary.props

```diff
@@ -181,6 +181,7 @@
       Microsoft.Extensions.Primitives;
       System.Diagnostics.EventLog;
       System.IO.Pipelines;
+      System.Net.Quic;
```

Comment:
@ViktorHofer I dont this as a part of the aspnetcore ref pack(preview 7) ? whats the reason behind including this ?

Reply:
It was part of the transport package so it should continue to be part of it. Check the transport package before the pkgproj to csproj migration: https://dnceng.visualstudio.com/public/_packaging?_a=package&feed=dotnet6-transport%40Local&package=Microsoft.AspNetCore.Internal.Transport&protocolType=NuGet&version=6.0.0-rc.1.21401.1&view=overview.

We carry this along in the transport package and aspnetcore then excludes the assembly.

Reply:
But here we are using this list to not update the assembly versions for the things which are part of the ref pack during servicing. As system.Net.Quic is not a part of the ref packs we should be able to update the assembly verison.
We could just manually add a project reference to the transport package to have this assembly.

Reply:
System.Net.Quic is already part of the Microsoft.NETCore.App shared framework: https://github.com/dotnet/runtime/blob/7b04bfc22950015374436c8998908487ecd5f7ba/src/libraries/NetCoreAppLibrary.props#L67. It doesn't matter if it's added to the package via a direct ProjectReference or via an entry in the AspNetCoreAppLibrary property.

Reply:
My only concern left here is that having it in the AspNetCoreAppLibrary property implies that its the part of the aspnetcore shared framework which it is not.

Reply:
Which definitely is a valid concern. Just don't want to again move things around before branching off.